Multisig/Cross-layer/App.swift

```diff
@@ -69,7 +69,5 @@ class App {
 
     let firebaseConfig = FirebaseConfig()
 
-    let intercomConfig = IntercomConfig()
```

Comment:
Why?

Reply:
It's not necessary. All calls to the intercom api are static.